src/tap_reporter.wren

```diff
@@ -0,0 +1,53 @@
+import "./capabilities" for Capabilities
+import "./stacktrace_report" for StackTraceReport
+import "./expect" for Expect
+
+StackTraceReport.withoutColors
+Expect.withoutColors
+
+class TapReporter {
+    construct new(name) {
+        _name = name
+        _fail = _skip = _success = _number = 0
+        _plan = false
+    }
+    start() {
+        System.print("TAP version 13")
```

Comment:
Let's make the version # a static class getter or hoist it outside the class entirely into a variable.